index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="assets/style/style.css">
+  <title>Cinema | Showcase</title>
+</head>
+<body>
+  <header class="header-bar">
+    <nav class="nav-bar">
+      <img class="humberger" src="assets/icons/menu.png" alt="menu" id="menubtn">
+      <ul class="menu" id="menu">
+        <div class="close-menu" id="closeMenu"></div>
+        <li><a href="index.html#">Home</a></li>
+        <li><a href="#programSection">Programs</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#newEvents">News</a></li>
+        <li><a href="#topMovies">Movies</a></li>
+        <li><a href="#ourPartners">Our partners</a></li>
+        <li class="pres"><button type="button" class="prettion">presentation</button></li>
+      </ul>
+    </nav>
+    <div class="logo">
+      <img src="assets/icons/cinema.png" alt="This is a cinema logo">
+    </div>
+  </header>
+   <main class="main">
+      <section class="welcome-events">
+        <div class="presnt">
+          <div class="welcome">
+            <p class="wlcm">Welcome to Cinema Showcase</p>
+            <div class="presentation">
+              <video class="vid" autoplay muted loop src="assets/img/wave.mp4"></video>
+              <p>Unlock Ultimate Movie Magic!</p>
+            </div>
+          </div>
+          <p class="welcome-paragraph">
+            Welcome to Cinema Showcase! Get ready to be entertained with classic and contemporary films, movie reviews, interviews, and more!
+          </p>
+        </div>
+        <div class="cine-events" id="newEvents">
+          
+        </div>
+      </section>
+      <section class="main-program" id="programSection">
+        <div class="programs-title">
+          <h3>Main Program</h3>
+          <hr>
+        </div>
+        <ul class="programs-list" id="allPrograms">
+        </ul>
+        <p class="more-prog"><a href="">See more programs ></a></p>
+      </section>
+      <section class="top-movies" id="topMovies">
+        <div class="top-movies-title">
+          <h3>Top Movies</h3>
+          <hr>
+        </div>
+        <ul class="top-movies-list" id="moviesList">
+          
+        </ul>
+      </section>
+      <section class="partners-section" id="ourPartners">
+        <div class="top-movies-title">
+          <h3>Our partners</h3>
+          <hr>
+        </div>
+        <ul class="partner-list" id="partnerList">
+                    
+        </ul>
+      </section>
+   </main>
+   <footer class="footer">
+    <div class="copy-logo">
+      <div class="p-logo">
+        <img src="assets/icons/cinema.png" alt="This is a cinema logo">
+      </div>
+      <div class="s-logo">
+        <img src="assets/icons/cinema.png" alt="This is a cinema logo">
+      </div>
```

Comment:
- [ ] Kindly try to adjust your footer section for both the `home page` and the  `about section`  to replicate the original design. This includes the background colors and elements.

![image](https://user-images.githubusercontent.com/98453979/225989262-aef5e826-e9c7-47b1-8e38-49a7864a2d43.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/225989529-00f3cf0b-3498-4a19-95f2-1f310c903717.png)

Reply:
Done 💯